Singularity.def

```diff
@@ -0,0 +1,25 @@
+Bootstrap: docker
```

Comment:
This sort of file shouldn't live at the root of the project. Our Docker building script is in `scripts`. Can you move this there or create a new appropriate directory for it?

Reply:
Fixed